### DISCUSSION TITLE
Chrome and Vivaldi extension buttons leads to dead links

### DISCUSSION THREAD

**Discussion by Bohaska** [subhra74/xdm#723] [Q&A]
I am unable to download the browser extensions because the links give me 404 errors.

**Comment by ahmad-bravo** [comment]
Extension Links for Chrome. Chromium and Opera are not working. All of them are giving 404 errors

**Comment by chuckyice** [comment]
getting 404 too with chromium

**Comment by oasaph** [comment]
For those who couldn't find the extension, you still can clone the repository and load the directory directly on chromium based browsers (I don't know a method for those that aren't)
1. Clone the repository onto your computer or download it as zip and unpack it
![image](https://user-images.githubusercontent.com/20308482/195962432-acf039a4-095a-4388-bda6-3fe1e273b1a5.png)
1. Go to the extensions configuration page (_e.g._ `chrome://extensions/`)  and enable developer mode 
![image](https://user-images.githubusercontent.com/20308482/195962279-8b40e45d-f313-4b7a-b59c-abac024125d5.png)
2. Click on `load unpacked`, navigate to the directory `clone-parent/xdm/app/xdm-browser-monitor/chrome` and click select folder 
![image](https://user-images.githubusercontent.com/20308482/195962464-8cc8596c-9ea2-4eab-9536-8b7b7628d090.png)
3. You're done
![image](https://user-images.githubusercontent.com/20308482/195962475-cd96388f-d832-48c2-88a0-d5933a723487.png)

**Reply by Primarch-Alpharius** [reply]
Worked like a charm, thank you.

**Reply by dumbfuckjuic** [reply]
thx make a vid on this or i will do it lol

**Reply by GoofyLackin69** [reply]
I've tried your method and it seems there's an error about the extension
![image](https://github.com/subhra74/xdm/assets/167872232/4bd127dc-81e6-4576-850d-b0ec966a5dc1)

here's the image showing some code that I have no idea what it is about. how to fix it?

oh FYI I'm using OperaGX

**Comment by subhra74** [comment]
Please install latest version from https://github.com/subhra74/xdm-experimental-binaries/releases/tag/8.0.11-beta.
The browser extension must be installed from XDM browser monitoring screen. Instructions to install the extension is provided there as well.
